Formula/planck.rb

```diff
@@ -21,8 +21,39 @@ class Planck < Formula
   depends_on "icu4c"
   depends_on "libzip"
 
+  uses_from_macos "vim" => :build # for xxd
+  uses_from_macos "curl"
+  uses_from_macos "libffi"
+
+  on_linux do
+    depends_on "gcc"
+    depends_on "glib"
+    depends_on "pcre"
+    depends_on "webkitgtk"
+  end
+
+  fails_with gcc: "5"
+
+  # Apply upstream commit to fix issue with GNU sed.  Remove with next release.
+  patch do
+    url "https://github.com/planck-repl/planck/commit/f1f21bf9eb4cfca6312ff0117f75d3b38164b43d.patch?full_index=1"
+    sha256 "787ddf6fb1cfea1d70fa18a6f7b292579ea1ffbf1f437f1f775e3330d2b8d36c"
+  end
+
   def install
     ENV["JAVA_HOME"] = Formula["openjdk"].opt_prefix
+
+    unless OS.mac?
+      ENV.prepend_path "PATH", Formula["openjdk"].opt_bin unless OS.mac?
+
+      # The webkitgtk pkg-config .pc file includes the API version in its name (ex. javascriptcore-4.1.pc).
+      # We extract this from the filename programmatically and store it in javascriptcore_api_version
+      # and make sure planck-c/CMakeLists.txt is updated accordingly.
+      javascriptcore_pc_file = (Formula["webkitgtk"].share/"pkgconfig").glob("javascriptcoregtk-*.pc").first
```

Comment:
```suggestion
      javascriptcore_pc_file = (Formula["webkitgtk"].lib/"pkgconfig").glob("javascriptcoregtk-*.pc").first
```

Reply:
Ah I think I messed this up because in the other formula I copied this code from, it was an oddball one that put the `pkg-config` files in `share`.  I'm going to do a quick check in Linux though to be sure.